app/models/like.rb

```diff
@@ -0,0 +1,5 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User',
+                      foreign_key: 'author_id'
+  belongs_to :post, counter_cache: :likes_count
+end
```

Comment:
- [ ] Like model should have a method that updates the likes counter for a post: 

```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```